### DISCUSSION TITLE
Multiple application definition in folders

### DISCUSSION THREAD

**Discussion by lfdominguez** [argoproj/argo-cd#6121] [Q&A] [answered]
Hi, I have this tree structure for a deploy project
```
root
  |-------app/
             |---- deploy1/ (Raw Kubernetes YAML deploy)
             |---- deploy2/ (Kustomize deploy)
  |-------deps/
             |---- dep1/ (Helm deploy)
             |---- dep2/ (Kustomize deploy)
```
In ArgoCD i see the Recursive directory option... with this option this structure is supported???

**Comment by wtam2018** [comment, marked as answer]
Yes,.  You can use ApplicationSet's GitGenerator to watch your git directory and deploy multiple applications automatically.  More details here:   https://argocd-applicationset.readthedocs.io/en/stable/Generators/#git-generator-directories

**Reply by lfdominguez** [reply]
Thanks... exactly that... a RTFM to me....
